packages/twenty-front/src/modules/ui/layout/page/PageHeader.tsx

```diff
@@ -71,18 +71,41 @@ const StyledTopBarButtonContainer = styled.div`
   margin-right: ${({ theme }) => theme.spacing(1)};
 `;
 
+const StyledSkeletonLoader = styled.div`
```

Comment:
Could you use what we already implemented in `DropdownMenuSkeletonItem` ? With `react-loading-skeleton`

Reply:
Hello @lucasbordeau, sorry, didn't get you. need clarification from you.

Do you want me to build skeleton using react-loading-skeleton or you want me to use DropdownMenuSkeletonItem in place of StyledSkeletonLoader

Reply:
Sure, no don't use directly because it's a MenuItem and should be only used in dropdowns.

You can just copy the implementation in your component and adapt it to your needs.